test/js/specs/happy.js

```diff
@@ -2,125 +2,454 @@ const path = require('path')
+      documentUpload.crossDeviceSubMessage.isDisplayed()
+    })
+
+    it('should display cross device arrow presence', async () => {
+      documentUpload.crossDeviceArrow.isDisplayed()
+    })
+
+    it('should display uploader icon presence', async () => {
+      documentUpload.uploaderIcon.isDisplayed()
+    })
+
+    it('should display uploader button', async () => {
+      const uploaderBtnText = documentUpload.uploaderBtn.getText()
+      expect(uploaderBtnText).to.equal(documentUploadLocale["capture"]["upload_file"])
+      documentUpload.uploaderBtn.isDisplayed()
+    })
+
+    it('should upload a passport and verify UI elements', async () => {
       driver.get(localhostUrl)
-      welcome.getPrimaryBtn().click()
-      documentSelection.getPassport().click()
+      welcome.primaryBtn.click()
+      documentSelection.passportIcon.click()
+      const passportTitleText =  documentUpload.title.getText()
+      expect(passportTitleText).to.equal(documentUploadLocale["capture"]["passport"]["front"]["title"])
+      documentUpload.title.isDisplayed()
+      const passportInstructionMessage = documentUpload.uploaderInstructionsMessage.getText()
+      expect(passportInstructionMessage).to.equal(documentUploadLocale["capture"]["passport"]["front"]["instructions"])
+      documentUpload.uploaderInstructionsMessage.isDisplayed()
+      const input = documentUpload.getUploadInput()
+      input.sendKeys(path.join(__dirname, '../../features/helpers/resources/passport.jpg'))
+      waitForUploadToFinish
+      const checkReadabilityText = documentUpload.title.getText()
+      expect(checkReadabilityText).to.equal(documentUploadLocale["confirm"]["document"]["title"])
+      const makeSureClearDetailsMessage = documentUploadConfirmation.makeSureClearDetailsMessage.getText()
+      expect(makeSureClearDetailsMessage).to.equal(documentUploadLocale["confirm"]["passport"]["message"])
+      documentUploadConfirmation.makeSureClearDetailsMessage.isDisplayed()
+    })
+
+    it('should upload driving licence and verify UI elements', async () => {
+      driver.get(localhostUrl)
+      welcome.primaryBtn.click()
+      documentSelection.drivingLicenceIcon.click()
+      const frontOfDrivingLicenceTitle = documentUpload.title.getText()
+      documentUpload.title.isDisplayed()
+      expect(frontOfDrivingLicenceTitle).to.equal(documentUploadLocale["capture"]["driving_licence"]["front"]["title"])
+      const frontOfDrivingLicenceInstructionMessage = documentUpload.uploaderInstructionsMessage.getText()
+      expect(frontOfDrivingLicenceInstructionMessage).to.equal(documentUploadLocale["capture"]["driving_licence"]["front"]["instructions"])
+      documentUpload.uploaderInstructionsMessage.isDisplayed()
+      const uploadFront = documentUpload.getUploadInput()
+      uploadFront.sendKeys(path.join(__dirname,'../../features/helpers/resources/uk_driving_licence.png'))
```

Comment:
the path should really be abstracted away, makes it noisy to rest the test otherwise